src/umap.cpp

```diff
@@ -57,8 +57,7 @@ class _umap {
         void* segment_address;
         size_t segment_length;
         int backingfile_fd;
```

Comment:
Okay, I will fix this after the merge since I am already working on evict_page and pagefault_event()